src/components/input/SelectNext.tsx

```diff
@@ -474,8 +476,25 @@ function SelectMain<T>({
   );
 }
 
-SelectMain.displayName = 'SelectNext';
+export const SelectNext = Object.assign(SelectMain, {
+  Option: SelectOption,
+  displayName: 'SelectNext',
+});
+
+export type SelectProps<T> = BaseSelectProps & SingleValueProps<T>;
+
+export const Select = Object.assign(
+  function <T>(props: SelectProps<T>) {
+    return <SelectNext<T> {...props} />;
+  },
+  { Option: SelectOption, displayName: 'Select' },
+);
 
-const SelectNext = Object.assign(SelectMain, { Option: SelectOption });
+export type MultiSelectProps<T> = BaseSelectProps & MultiValueProps<T>;
 
-export default SelectNext;
+export const MultiSelect = Object.assign(
```

Comment:
I choose `MultiSelect` over `SelectMulti` for consistency with `SingleValueProps` and `MultiValueProps`, but the latter allows for a nicer grouping of symbols when sorted alphabetically.